monkey-runner/sample/build.gradle

```diff
@@ -0,0 +1,22 @@
+task wrapper(type: Wrapper) {
+    gradleVersion = versions.gradleWrapperVersion
+}
+
+buildscript {
+    apply from: '../dependencies.gradle'
```

Comment:
same as [the comment above](https://github.com/novoda/spikes/pull/144/files#r73426394)